pkg/registry/storage/csidriver/strategy_test.go

```diff
@@ -167,21 +158,6 @@ func TestCSIDriverPrepareForCreate(t *testing.T) {
 					t.Errorf("VolumeLifecycleModes not stripped: %v", csiDriver.Spec)
 				}
 			}
-			if test.withServiceAccountToken {
-				if csiDriver.Spec.TokenRequests == nil {
```

Comment:
Should we keep the assertion that the fields are not dropped?

Reply:
in any case we no longer drop fields so no need to assert fields are not dropped.

Reply:
In the base CSIDriver object input to the test, can we set some values to these fields? That way we test that this method no longer mutates those fields.

Reply:
we already set. line 133-134.